coderd/workspaces.go

```diff
@@ -43,6 +43,19 @@ var (
 	errDeadlineBeforeStart = xerrors.New("new deadline must be before workspace start time")
 )
 
+// @Summary Get workspace metadata
+// @ID get-workspace
+// @Security CoderSessionToken
+// @Produce json
+// @Tags Workspaces
+// @Param id path string true "Workspace ID"
+// @Param include_deleted query string false "Include deleted"
```

Comment:
Since this is for docs the query param should have a better description, like "Return data instead of 404 if the workspace is deleted"

Reply:
Fixed